storybook/storyLoader.js

```diff
@@ -1,89 +1,92 @@
-  require("../src/lib/Components/Artist/__stories__/ArtistHeader.story.tsx")
-  require("../src/lib/Components/Bidding/Components/__stories__/Checkbox.story.tsx")
-  require("../src/lib/Components/Bidding/Components/__stories__/Input.story.tsx")
-  require("../src/lib/Components/Bidding/Components/__stories__/Markdown.story.tsx")
-  require("../src/lib/Components/Bidding/Components/__stories__/Timer.story.tsx")
-  require("../src/lib/Components/Bidding/Screens/__stories__/BidResult.story.tsx")
-  require("../src/lib/Components/Bidding/Screens/__stories__/RegistrationResult.story.tsx")
-  require("../src/lib/Components/Bidding/__stories__/BidFlow.story.tsx")
-  require("../src/lib/Components/Buttons/__stories__/Buttons.story.tsx")
-  require("../src/lib/Components/Consignments/__stories__/BottomAligned.story.tsx")
-  require("../src/lib/Components/Consignments/__stories__/Consignments.story.tsx")
-  require("../src/lib/Components/Consignments/__stories__/ImageSelection.story.tsx")
-  require("../src/lib/Components/Consignments/__stories__/Search.story.tsx")
-  require("../src/lib/Components/Consignments/__stories__/Style.story.tsx")
-  require("../src/lib/Components/Consignments/__stories__/Todo.story.tsx")
-  require("../src/lib/Components/Inbox/ActiveBids/__stories__/ActiveBids.story.tsx")
-  require("../src/lib/Components/Inbox/Conversations/Preview/__stories__/InvoicePreview.story.tsx")
-  require("../src/lib/Components/Inbox/Conversations/__stories__/ArtworkPreview.story.tsx")
-  require("../src/lib/Components/Inbox/Conversations/__stories__/Avatar.story.tsx")
-  require("../src/lib/Components/Inbox/Conversations/__stories__/Composer.story.tsx")
-  require("../src/lib/Components/Inbox/Conversations/__stories__/ConversationSnippet.story.tsx")
-  require("../src/lib/Components/Inbox/Conversations/__stories__/Inbox.story.tsx")
-  require("../src/lib/Components/Inbox/Conversations/__stories__/Message.story.tsx")
-  require("../src/lib/Components/Inbox/Conversations/__stories__/ZeroStateInbox.story.tsx")
-  require("../src/lib/Components/States/__stories__/ZeroState.story.tsx")
-  require("../src/lib/Components/Text/__stories__/Typography.story.tsx")
-  require("../src/lib/Components/__stories__/Modal.story.tsx")
-  require("../src/lib/Components/__stories__/DottedLine.story.tsx")
-  require("../src/lib/Containers/__stories__/Artist.story.tsx")
-  require("../src/lib/Containers/__stories__/Gene.story.tsx")
-  require("../src/lib/Containers/__stories__/Inquiry.story.tsx")
-  require("../src/lib/Containers/__stories__/Sale.story.tsx")
-  require("../src/lib/Scenes/Favorites/__stories__/Favorites.story.tsx")
-  require("../src/lib/Scenes/Home/Components/Sales/Components/__stories__/LotsByFollowedArtists.story.tsx")
-  require("../src/lib/Scenes/Home/__stories__/Home.story.tsx")
-  require("../src/lib/Scenes/Settings/__stories__/Settings.story.tsx")
+  require('../src/lib/Components/Artist/__stories__/ArtistArticles.story.tsx');
```

Comment:
Is this an intentional formatting change? I thought we preferred double quotes.

Reply:
I'm not sure why these changes came from, I thought there was a change in the storybook settings so I just added them as well. I just removed and this should be good.

Reply:
FYI this fine gets autogenerated by Yarn based on [this config](https://github.com/artsy/emission/blob/bd7a8b1be56467f40f9aa1e9403a32a45cd8025e/package.json#L174-L178) but sometimes Prettier messes with the quote formats.